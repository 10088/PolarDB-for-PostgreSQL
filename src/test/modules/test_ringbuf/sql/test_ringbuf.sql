CREATE EXTENSION test_ringbuf;

-- See README for explanation of arguments:
SELECT test_ringbuf();
