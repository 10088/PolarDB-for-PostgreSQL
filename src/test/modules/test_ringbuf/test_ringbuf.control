comment = 'Test code for ringbuf'
default_version = '1.0'
module_pathname = '$libdir/test_ringbuf'
relocatable = true
