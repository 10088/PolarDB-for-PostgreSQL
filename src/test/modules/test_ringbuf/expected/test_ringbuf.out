CREATE EXTENSION test_ringbuf;
-- See README for explanation of arguments:
SELECT test_ringbuf();
 test_ringbuf 
--------------
 
(1 row)

